moped-database/migrations/1668275290138_create_canceled_on_hold_proj_phases/up.sql

```diff
@@ -0,0 +1,33 @@
+-- insert an on hold phase for projects which have an on hold status
+DO $$
+DECLARE
+    project_id int;
+BEGIN
+    FOR project_id IN
+    SELECT
+        mp.project_id
+    FROM
+        moped_project mp
+    WHERE
+        current_status = 'on hold' LOOP
+            INSERT INTO moped_proj_phases (project_id, phase_id, is_current_phase, completion_percentage, completed)
+                VALUES(project_id, 14, TRUE, 0, FALSE);
+        END LOOP;
+    END$$;
+
+-- insert a canceled project phase for projects which have a canceled status
+DO $$
+DECLARE
+    project_id int;
+BEGIN
+    FOR project_id IN
+    SELECT
+        mp.project_id
+    FROM
+        moped_project mp
+    WHERE
+        current_status = 'canceled' LOOP
+            INSERT INTO moped_proj_phases (project_id, phase_id, is_current_phase, completion_percentage, completed)
+                VALUES(project_id, 15, TRUE, 0, FALSE);
+        END LOOP;
```

Comment:
Fancy!! TIL! This is going in my bag of tricks. 🧳

Reply:
+1 

when i found a code example for this task i was like "I DO DECLARE this is too easy"

Reply:
https://tenor.com/view/i-do-declare-king-of-the-hill-gif-13278066